homeassistant/components/localip/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""Get the local IP (eth0 equivalent) of the Home Assistant instance."""
+import homeassistant.util as hass_util
+
+DOMAIN = "localip"
```

Comment:
The integration should be renamed. 

```suggestion
DOMAIN = "local_ip"
```

Reply:
I don't have any sentimental attachment either way 😄 but can I ask why just for general knowledge?

Reply:
We prefer to separate words with underscore in names.

Reply:
Fixed